webapp/channels/src/components/admin_console/team_channel_settings/convert_confirm_modal.tsx

```diff
@@ -101,9 +113,14 @@ const messages = defineMessages({
     toPrivateMessage: {
         id: 'admin.team_channel_settings.convertConfirmModal.toPrivateMessage',
```

Comment:
even this message is now changed then but the en.json isnt updated with this change

Reply:
Done!